sns_ik_lib/CMakeLists.txt

```diff
@@ -30,29 +30,46 @@ catkin_package(
   LIBRARIES sns_ik
 )
 
-include_directories(include ${catkin_INCLUDE_DIRS} ${EIGEN3_INCLUDE_DIRS} ${orocos_kdl_INCLUDE_DIRS})
+include_directories(include utilities ${catkin_INCLUDE_DIRS} ${EIGEN3_INCLUDE_DIRS} ${orocos_kdl_INCLUDE_DIRS})
 link_directories(${orocos_kdl_LIBRARY_DIRS})
-install(DIRECTORY include/ DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
 
+# library for private utilities
+add_library(sns_ik_util
+            utilities/sns_ik_math_utils.cpp)
+target_link_libraries(sns_ik_util)
```

Comment:
Hmmmm. This linking is missing things to be linked to. You can likely remove line 39.

Reply:
That is unless you actually should be linking against things like `${catkin_LIBRARIES}`...

Reply:
Looks like everything works fine after deleting line 39.